main.scss

```diff
@@ -20,3 +20,7 @@
 @import "src/scss/field-errors";
 @import "src/scss/field-valid";
 @import "src/scss/global-and-section-messages";
+
+// Set silent mode back on again to avoid it being output twice
+$o-forms-is-silent: true;
+$o-ft-forms-is-silent: true;
```

Comment:
Origami policy is to use CSS optimiser to remove code duplication after the sass is built. We should stick to that policy until it changes (I don't object to having a discussion about changing the policy, but consistency and compliance trumps experimentation)